src/rpc/RPCHelpers.cpp

```diff
@@ -1350,6 +1350,19 @@ parseTaker(boost::json::value const& taker)
         return Status{RippledError::rpcBAD_ISSUER, "invalidTakerAccount"};
     return *takerID;
 }
+
+ripple::Issue
+parseIssue(boost::json::value const& issue)
+{
+    Json::Value jv;
+    if (issue.as_object().contains(JS(issuer)))
```

Comment:
"issue" could be a value instead of the object. Then jv will be empty, the ripple::issueFromJson will handle this: https://github.com/XRPLF/rippled/blob/90d463b925c377878d3abdb27eb3a1530782a668/src/ripple/protocol/impl/Issue.cpp#L80

Reply:
i think what will happen is `issue.as_object()` will result in a crash.

Reply:
You are right. because the validation check it first, so here we can guarantee it is object.

Reply:
Yea, the problem is that before we could not call this function/lambda from a new place but now in the future someone could call it without checking it's object first.

Reply:
Yea . Have fixed it.